client/client.lua

```diff
@@ -9,6 +9,7 @@ local closestScenes = {}
 
 local creationLaser = false
 local deletionLaser = false
+local permissionLevel = nil
```

Comment:
```suggestion
local permissionLevel = 'user'
```
Lets default this to 'user'.